frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -666,6 +667,19 @@ export const ResourceYAMLEditor: React.FC<ResourceYAMLEditorProps> = require('@c
 export const ResourceEventStream: React.FC<ResourceEventStreamProps> = require('@console/internal/components/events')
   .WrappedResourceEventStream;
 
+/**
+ * A component to change the document title of the page.
+ * @example
+ * ```tsx
+ * <Title>My Page Title</Title>
+ * ```
+ * This will change the title to "My Page Title · [Product Name]"
+ *
+ * @param {TitleProps} props - The props for the Title component
+ * @param {TitleProps['string']} children - The title to display
+ */
+export const Title: React.FC<TitleProps> = require('@console/shared/src/components/title/Title').Title;
```

Comment:
It would be nice to use this (once) somewhere in the dynamic demo plugin, what do you think?